.github/workflows/build.yaml

```diff
@@ -0,0 +1,25 @@
+name: Build
+on: [push]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        node-version: [12.x]
+
+    steps:
+      - uses: actions/checkout@v1
+      - name: Use Node.js ${{ matrix.node-version }}
+        uses: actions/setup-node@v1
+        with:
+          node-version: ${{ matrix.node-version }}
+      - name: build
+        run: |
+          npm ci
+          npm run format
```

Comment:
I don't really like the idea of the format in a CI.
I prefer that here only be verified that it is formatted correctly

Reply:
Oops. Thanks will fix